src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/ccd/CcdEventProcessorTest.java

```diff
@@ -43,33 +40,8 @@ class CcdEventProcessorTest {
     @Mock
     private ObjectMapper mapper;
 
-    @Mock
-    private LaunchDarklyFeatureFlagProvider featureFlagProvider;
-
     private CcdEventProcessor processor;
 
-    @Test
-    void should_not_trigger_handlers_when_feature_flag_is_false() throws JsonProcessingException {
```

Comment:
Should we be removing this test when the assertion is to do a particular thing based on the feature flag? I.e. removing the flag should be sufficient and the test should execute with the expected R2 journey.

Reply:
We have removed the logic of R1 route code as part of this changes, so R1 tests (when_feature_flag_disabled) have to be removed too.